src/realm/sync/client.hpp

```diff
@@ -13,6 +13,7 @@
 #include <realm/util/functional.hpp>
 #include <realm/util/future.hpp>
 #include <realm/sync/client_base.hpp>
+#include <realm/sync/socket_provider.hpp>
```

Comment:
Do we need this include? I don't see anything that uses it.

Reply:
removed.